src/main/java/com/cairn/waypoint/dashboard/entity/HomeworkResponseLinkedProtocol.java

```diff
@@ -16,21 +19,22 @@
 
 @Data
 @Entity
-@SuperBuilder
 @NoArgsConstructor
 @AllArgsConstructor
 @SQLRestriction("active=1")
-@EqualsAndHashCode(callSuper = true, exclude = {"homeworkTemplate", "homeworkQuestion"})
-@Table(name = "homework_template_linked_homework_question")
-public class HomeworkTemplateLinkedHomeworkQuestion extends BaseEntity {
-
-  @ManyToOne
-  @JoinColumn(name = "homework_template_id", nullable = false)
-  private HomeworkTemplate homeworkTemplate;
-
-  @JoinColumn(name = "homework_question_id", nullable = false)
-  @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.MERGE)
-  private HomeworkQuestion homeworkQuestion;
-
-  private Integer ordinalIndex;
-}
+@Table(name = "protocol_linked_homework_response")
+public class HomeworkResponseLinkedProtocol {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
```

Comment:
Same, let's remove this as it will result in bloatware. There isn't a usage that I can foresee and we will need to maintain it longterm if it is left in now